pyre/bench/synth/unspecialized_long_math_zip_paths.py

```
@@ -100,11 +131,11 @@ def f_ldexp(n):
-print("zip       ", f_zip(N))
-print("zip_strict", f_zip_strict(N))
-print("cmp_long  ", f_cmp_long_int(N))
-print("truediv   ", int(round(f_truediv_long(N) * 1e6)))
-print("pow       ", f_pow_long_int(N))
-print("shift     ", f_shift_long_int(N))
-print("frexp     ", int(round(f_frexp(N) * 1e6)))
-print("ldexp     ", int(round(f_ldexp(N) * 1e6)))
+print("zip       ", f_zip(N_ZIP))
+print("zip_strict", f_zip_strict(N_ZIP_STRICT))
+print("cmp_long  ", f_cmp_long_int(N_CMP))
+print("truediv   ", int(round(f_truediv_long(N_TRUEDIV) * 1e6)))
```

Ruff (0.16.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)